src/pal/src/exception/seh-unwind.cpp

```diff
@@ -141,7 +141,7 @@ static void UnwindContextToWinContext(unw_cursor_t *cursor, CONTEXT *winContext)
 #elif defined(_ARM_)
     unw_get_reg(cursor, UNW_REG_SP, (unw_word_t *) &winContext->Sp);
     unw_get_reg(cursor, UNW_REG_IP, (unw_word_t *) &winContext->Pc);
-    winContext->Pc &= ~0x1;
+    winContext->Pc |= 0x1;
```

Comment:
Ah, I have not realized that we were explicitly clearing the bit. So maybe the value extracted from the cursor has the bit set correctly?